dev/com.ibm.ws.security.jwtsso_fat/fat/src/com/ibm/ws/security/jwtsso/fat/ConfigAttributeTests.java

```diff
@@ -0,0 +1,483 @@
+
+        WebClient wc = new WebClient();
+        Page response = invokeUrl(wc, protectedUrl); // get back the login page
+        validationUtils.validateResult(response, ACTION_INVOKE_PROTECTED_RESOURCE, expectations);
+
+        response = performLogin(response); // should fail and we should get login page again
+
+        String responseStr = response.getWebResponse().getContentAsString();
+        boolean check = responseStr.contains("Form Login Page");
+        assertTrue("Did not receive login page on failure", check);
+
+        String errorMsg = server.waitForStringInLogUsingMark("CWWKS6030E", 100);
+        assertFalse("Did not find expected error message CWWKS6030E in log", errorMsg == null);
+
+    }
+
+    /**
+     * Test the fallbackToLtpa attribute.
+     * Specify an invalid builder, includeLtpa, and fallBackToLtpa. There should be no jwt cookie present,
+     * there should be an ltpa cookie present, and because fallback is enabled, we should be able to access
+     * the resource.
+     *
+     * @throws Exception
+     */
+    @Test
+    @Mode(TestMode.LITE)
+    @AllowedFFDC({ "com.ibm.websphere.security.jwt.InvalidBuilderException",
+                   "com.ibm.ws.security.jwt.internal.JwtTokenException",
+                   "com.ibm.websphere.security.WSSecurityException" })
+    public void test_fallbackToLtpaTrue() throws Exception {
+        reconfigServer("server_testfallbacktoltpatrue.xml");
+        Expectations expectations = new Expectations();
+        expectations.addExpectations(getSuccessfulLoginPageExpectations(ACTION_INVOKE_PROTECTED_RESOURCE));
+
+        String protectedUrl = "http://" + server.getHostname() + ":" + server.getHttpDefaultPort() + "/formlogin/SimpleServlet";
+
+        WebClient wc = new WebClient();
+        Page response = invokeUrl(wc, protectedUrl); // get back the login page
+        validationUtils.validateResult(response, ACTION_INVOKE_PROTECTED_RESOURCE, expectations);
+
+        response = performLogin(response);
+
+        String responseStr = response.getWebResponse().getContentAsString();
+        boolean check = responseStr.contains("SimpleServlet");
+        assertTrue("did not access protected resource", check);
+        assertTrue("did not find expected ltpa cookie", responseStr.contains("cookie: LtpaToken2"));
+        assertFalse("found unexpected jwt cookie", responseStr.contains("cookie: jwtToken"));
+    }
+
+    /**
+     * Test the setCookieSecureFlag attribute. Use the default setting (true),
+     * and inspect cookie to see that it happened.
+     *
+     * @throws Exception
+     */
+    @Test
+    @Mode(TestMode.LITE)
+    public void test_cookieSecureTrue_httpOnlyTrue() throws Exception {
+        reconfigServer("server_testcookiesecure.xml");
+        ArrayList<String> ignoredErrors = new ArrayList<String>();
+        ignoredErrors.add("CWWKS9127W");
```

Comment:
`CWWKS9127W_JWT_COOKIE_SECURITY_MISMATCH`. Could also update this test to use HTTPS to avoid emitting this message, unless you're specifically trying to test that particular scenario.